packages/health-check/src/checks/verbose.js

```diff
@@ -1033,26 +1047,36 @@ function parseHeaderString(header) {
 }
 
 // skylinkVerification verifies a skylink against provided information.
-function skylinkVerification(done, { name, skylink, bodyHash, headers }) {
+function skylinkVerification(done, { name, skylink, bodyHash, headers, statusCode }, { redirects } = {}) {
   const time = process.hrtime();
 
   // Create the query for the skylink
-  const query = `${process.env.PORTAL_URL}/${skylink}?nocache=true`;
+  const query = `${process.env.PORTAL_URL}/${skylink}`;
 
   // Get the Skylink
   superagent
     .get(query)
+    .set("cookie", "nocache=true")
```

Comment:
move `?nocache=true` from url to cookie because it would be harder to assert proper `location` header response with `?nocache=true` in it